idaes/generic_models/properties/core/state_definitions/FpcTP.py

```diff
@@ -22,6 +22,8 @@
                         EnergyBalanceType)
 from idaes.generic_models.properties.core.generic.utility import \
     get_bounds_from_config
+from .electrolyte_states import \
```

Comment:
Why did you choose not to include the whole path? 

I don't think it really matters, but its also not consistent with the other imports.

Reply:
@TimBartholomew The only reason is to keep the import line short.